doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -1325,6 +1325,41 @@ Recalculate the UNF value of a dataset version, if it's missing, by supplying th
 
   POST http://$SERVER/api/admin/datasets/integrity/{datasetVersionId}/fixmissingunf
 
+Dataset Validation
+~~~~~~~~~~~~~~~~~~
+
+Validate the dataset and its components (DatasetVersion, FileMetadatas, etc.) for constraint violations::
+
+  GET http://$SERVER/api/admin/validate/dataset/{datasetId}
```

Comment:
Lately, like in 50c5c2d, I've been favoring this form instead:

![Screen Shot 2019-04-26 at 11 59 27 AM](https://user-images.githubusercontent.com/21006/56820774-c2506d80-681a-11e9-94a7-a1dc5111f7e2.png)

This is for two reasons:

- Full curl commands you can copy and paste are easier for users. @donsizemore and I chatted about this at http://irclog.iq.harvard.edu/dataverse/2019-03-15#i_88140
- I don't like it when "http" is hard coded when sometimes "https" should be used instead.

Reply:
I'll add "curl" there - but the only reason I checked it in like that was that all the other examples in that section of the guide were just "GET" or "POST"; so I assumed that was still the convention.

Reply:
Agree about curl. As for the hard-coded "http" - it's kind of makes sense in this case; because this is under the "/admin" api, and our recommended setup is to restrict it to localhost, and access it under "http://localhost:8080". 
But yes, I'll change it to the notation above.